### PR TITLE
300 rounded time

### PR DESCRIPTION
**Pull request by Flaburgan**

### CODE REVIEW

File: axolotl-web/src/components/Message.vue [hunk 1]
```diff
@@ -176,16 +184,10 @@ export default {
       return true
     },
     humanifyDate(inputDate){
-      moment.locale(this.$language.current);
```
Comment (Flaburgan) [inline axolotl-web/src/components/Message.vue:179]: I don't know why, but this doesn't look needed from my side, I removed it and the time displayed are still in French.

File: axolotl-web/src/components/Message.vue [hunk 2]
```diff
@@ -176,16 +184,10 @@ export default {
...
-        if(min == 0) return "now"
-        return moment().diff(date, 'minutes') +" min"
-      }
-      var hours = moment().diff(date, 'hours')
-      if(hours <24) return hours + " h"
-      return date.format("DD. MMM");
+      return new moment(inputDate).format('lll');
+    },
+    humanifyDateFromNow(inputDate){
+      return new moment(inputDate).fromNow();
```
Comment (Flaburgan) [inline axolotl-web/src/components/Message.vue:190]: Simplification! (And with this, better localization as we remove "now").

**Review by nanu-c — APPROVED** [review]

**Comment by nanu-c** [comment]
But, if it says 12 days ago and 24 days ago, maybe we need to switch the output at some point in the past

**Comment by Flaburgan** [comment]
That is maybe a good point, I don't know... Now the user can tap it to see the real date anyway :D

**Pull request merged**